src/services/playerAudioEvents.ts

```diff
@@ -356,4 +357,44 @@ module.exports = async () => {
     const isVideo = false
     handleBackgroundTimerInterval(isVideo)
   })
+
+  // Android Auto Handlers
+  if (Platform.OS === 'android') {
+    PVAudioPlayer.addEventListener(Event.RemotePlayId, (e) => {
+      handlePlayRemoteMediaId(e.id)
+    })
+    PVAudioPlayer.addEventListener(Event.RemoteSkip, (e) => {
+      PVAudioPlayer.skip(e.index).then(() => PVAudioPlayer.play())
+    })
+    PVAudioPlayer.addEventListener(Event.RemoteBrowse, (e) => {
+      handleAABrowseMediaId(e.mediaId)
+    })
+    PVAudioPlayer.addEventListener(Event.RemotePlaySearch, (e) => {
+      // TODO: handle this
+      console.warn(e, 'not handled')
+    })
+
+    // TODO: handle skip next/previous via customActions in android auto
+    PVAudioPlayer.addEventListener(Event.RemoteCustomAction, (e) => {
+      console.log('Event.RemoteCustomAction', e)
+      switch (e.customAction) {
+        case 'customSkipPrev':
+          playerPlayPreviousChapterOrReturnToBeginningOfTrack()
+          break
+        case 'customSkipNext':
+          playerPlayNextChapterOrQueueItem()
+          break
+        case 'customJumpForward':
+          const { jumpBackwardsTime } = getGlobal()
+          audioJumpBackward(jumpBackwardsTime)
+          break
+        case 'customJumpBackward':
+          const { jumpForwardsTime } = getGlobal()
+          audioJumpForward(jumpForwardsTime)
+          break
+        default:
+          break
+      }
+    })
+  }
```

Comment:
These event don't look specific to android auto and I'm worried they will be registered even when bluetooth headphones are connected. I think these all need to go into the onConnect function of PVCarPlay.android.ts. I know you removed that function but I'm not sure we should remove it after all and do all these even listener setups for android, there.

Reply:
all of these are actually android auto specific - the customXXXX is specifically for Android <13 devices where the jumpforward and backward buttons do not show on the android auto control screen; RemoteXXXX are only triggered via android auto; RemoteSearch is triggered via Google Voice Assistant